### ISSUE TITLE
React Native Upgrade 0.69 - Spike to investigate Bundled Hermes

### ISSUE THREAD

**Issue by derekblank** [wordpress-mobile/gutenberg-mobile#5043]
Investigate RN 0.69's [Bundled Hermes](https://reactnative.dev/blog/2022/06/21/version-069#bundled-hermes) feature and how it might affect or interact with the current mobile Hermes implementation.

- [x] Review RN's deep dive into the [Bundled Hermes architecture](https://reactnative.dev/architecture/bundled-hermes)
- [x] Review the current status of Hermes within Gutenberg Mobile
- [x] **WordPress-Android**: Determine how Bundled Hermes might affect how we publish binaries to [react-native-libraries-publisher](https://github.com/wordpress-mobile/react-native-libraries-publisher)
- [x] **WordPress-iOS**: Determine the path needed to enable Hermes on iOS
- [x] Add follow-up tasks as needed.

**Comment by geriux** [comment]
Hey @derekblank 👋 I created 3-gh-wordpress-mobile/react-native-mirror-publisher to track any investigations or needed changes for `0.69` since this would also be needed for upgrading the dependencies (it's currently a blocker). I also pinged someone that worked on that project to see if they have any thoughts about the required changes.

Feel free to post any findings related to the new structure of `hermes-engine` and `react-native` in that ticket if it's related to the `react-native-mirror-publisher` repo.

Thank you!
